api/server/router/image/image_routes.go

```diff
@@ -357,6 +360,7 @@ func (s *imageRouter) getImagesSearch(ctx context.Context, w http.ResponseWriter
 		if err := json.NewDecoder(authJSON).Decode(&config); err != nil {
 			// for a search it is not an error if no auth was given
 			// to increase compatibility with the existing api it is defaulting to be empty
+			logrus.Warn("The deamon is failed decoding \"X-Regstry-Auth\"")
```

Comment:
I think "Failed decoding the X-Registry-Auth field in the request header" is good enough.
@thaJeztah WDYT?

Reply:
ok :+1:

Reply:
Wondering if instead this should return an error instead of just logging a warning. After all, the request is invalid

Reply:
@thaJeztah I will update with error ! , can you please look into the docs changes also ?